src/Ledger/Gov.lagda

```diff
@@ -81,6 +82,40 @@ private variable
 \end{code}
 \emph{Functions used in the GOV rules}
 \begin{AgdaMultiCode}
+\begin{code}[hide]
+govActionPriority : GovAction → ℕ
+govActionPriority NoConfidence             = 0
+govActionPriority (UpdateCommittee _ _ _)  = 1
+govActionPriority (NewConstitution _ _)    = 2
+govActionPriority (TriggerHF _)            = 3
+govActionPriority (ChangePParams _)        = 4
+govActionPriority (TreasuryWdrl _)         = 5
+govActionPriority Info                     = 6
+
+groupGovActions : GovState → GovState
```

Comment:
This is unused

Reply:
Yes, it's unused now, but wouldn't it be useful if/when we want to make sure all gov actions in a GovState are in the right order? Well, I'll just delete it for now. We can always add it back in later if needed.